src/locale/locales/de/messages.po

```diff
@@ -636,32 +602,28 @@ msgstr "Überprüfe deinen Posteingang auf eine E-Mail mit dem Bestätigungscode
 
 #: src/view/com/modals/Threadgate.tsx:72
 msgid "Choose \"Everybody\" or \"Nobody\""
-msgstr ""
+msgstr "Wählen Sie \"Alle\" oder \"Niemand\""
 
 #: src/view/screens/Settings/index.tsx:697
 msgid "Choose a new Bluesky username or create"
-msgstr ""
+msgstr "Wähle einen neuen Bluesky-Benutzernamen oder erstelle"
 
 #: src/view/com/auth/server-input/index.tsx:79
 msgid "Choose Service"
 msgstr "Service wählen"
 
 #: src/screens/Onboarding/StepFinished.tsx:135
 msgid "Choose the algorithms that power your custom feeds."
-msgstr ""
+msgstr "Wählen Sie die Algorithmen, die Ihre benutzerdefinierten Feeds betreiben."
```

Comment:
"power" was translated with "betreiben", which means something like "operate a machine" or "you have a small business that sells orange juice".

It kinda works, but also sounds a bit weird. But I have no idea of a better word right now. Maybe someone else has.

Reply:
OK I'll leave this one open for now.

Reply:
maybe it's not the best option because it changes the word but how about using `Wählen Sie die Algorithmen, die Ihre benutzerdefinierten Feeds generieren.`

Reply:
> maybe it's not the best option because it changes the word but how about using `Wählen Sie die Algorithmen, die Ihre benutzerdefinierten Feeds generieren.`

I'm obviously no expert translator or anything, but my understanding is that the goal is to convey the intended meaning, even if you have to use a different word or expression to accomplish that.

Deep L translates the text you proposed as `Select the algorithms that generate your customised feeds.` which seems to fit the meaning of the original English phrase pretty closely imo. So I'll go with that, thanks 👍